src/shops/api/shops/__fixtures__/mountfield/product-detail.html

```
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html><body>
+<nav class="box-breadcrumb">
+<span class="box-breadcrumb__item">Domů</span>
+<span class="box-breadcrumb__item">Travní sekačky</span>
+<span class="box-breadcrumb__item">STIGA</span>
+</nav>
+<div class="list-products__item__in">
+  <a class="list-products__item__block" href="/akumulatorova-travni-sekacka-stiga-collector-548-ae-nh-7300003"></a>
+  <h2>Akumulátorová travní sekačka STIGA Collector 548 AE NH</h2>
+  <img src="https://example.com/img1.jpg" />
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)